client/src/sagas/actions.js

```diff
@@ -0,0 +1,30 @@
+import { auth, alert } from './types';
+
+const addAlert = (msg, type) => ({
+  type: alert.ADD_ALERT,
+  payload: { msg, type },
+});
+
+export const alertActions = {
+  addAlert,
+};
```

Comment:
🖍️  What benefit do we achieve by having the alert and auth actions in the same file under `sagas`??

Reply:
Yeah, they shouldn't be. I'm hoping @amichael497 can help me with ideas for organizing this stuff. But also I could take a crack at something better than this :#